karma-story-editor.config.cjs

```diff
@@ -68,9 +68,10 @@ module.exports = function (config) {
         served: true,
         nocache: false,
       },
+      // eslint-disable-next-line prettier/prettier, no-useless-escape
+      '^https:\/\/fonts.googleapis.com\/css*',
```

Comment:
Seeing `Pattern "/home/runner/work/web-stories-wp/web-stories-wp/^https:/fonts.googleapis.com/css*" does not match any file.` in the Karma logs now